makemkv: 1.14.0 -> 1.14.1

nyanloutre opened the pull request:
###### Motivation for this change

Previous URL is unreachable

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

nyanloutre committed:
makemkv: 1.14.0 -> 1.14.1

c0bw3b reviewed (changes requested):
There was #50437 already
Same remarks apply : it needs two URLs for both files to avoid breaking at every minor bump..

nyanloutre committed:
makemkv: using two URLs for src

This will prevent future failures due to the first URL being broken each
time a new version is released

See https://github.com/NixOS/nixpkgs/pull/50437#issuecomment-439329862
for details

nyanloutre commented:
@c0bw3b I just pushed a fix

c0bw3b commented:
@nyanloutre thank you for this
